src/FEValues/GeometryValues.jl

```diff
@@ -0,0 +1,84 @@
+struct GeometryValues{dMdξ_t, GIP, T}
+    M::Matrix{T}
+    dMdξ::Matrix{dMdξ_t}
```

Comment:
I was always wondering why we use Matrix/Vector instead of SMatrix/SVector here. Is there any good reason to not use it? If we want to keep the ability to resize these here, then we should just keep it as is. Not sure.

Reply:
I haven't thought of that. Would be interesting to benchmark if it makes a difference. But I suspect that the number of floats is too high in some cases to make sense, e.g. in 3d with 8 quad points for a Hexahedron, we have 8*(8*(1+3))=256 floats.

Reply:
Well, indeed a good point.